content/applications/marketing/marketing_automation/getting_started/target_audience.rst

```diff
@@ -0,0 +1,119 @@
+parameters, and further allows parameters to be clustered with *Any* or *All* statements, and a
+*node* adds another parameter to the branch.
+
+Every time a new branch is created, there are two options:
+
+- Either the records can match :guilabel:`ALL` criteria for the upcoming rules (an AND statement
+  where all criteria must match).
+
+- Or, the records can match :guilabel:`ANY` criteria for the upcoming rules (an OR statement where
+  only one of the criteria must match).
+
+To change between these two options, simply click the drop-down arrow icon in the green box and
+select :guilabel:`Any` or :guilabel:`All`.
+
+To add a node, click on the :guilabel:`+ (plus sign icon)` and to add another branch click on the
+:guilabel:`... (ellipses icon)`. To exclude a :guilabel:`node` (or a :guilabel:`branch`), click on
+:guilabel:`x (delete)`.
+
+.. image:: target_audience/marketing-filter-nodes.png
+   :align: center
+   :alt: The drop-down filter menu in the Marketing Automation application.
+
+Use cases
+=========
+
+The following scenarios are examples of some filters a marketing campaign might use.
+
+**Scenario**: Narrow the target down to new opportunities in the pipeline.
+
+While in Edit mode on a campaign template form, select the :guilabel:`Target` field, and click 
+:guilabel:`Search More` from the drop-down menu. Then, scroll down (or search for) 
+:guilabel:`Lead/Opportunity` and select it.
+
+Next, click :guilabel:`Add Filter` in the :guilabel:`Filter` field. Click on the default 
+:guilabel:`ID` filter option in the first portion of the filter equation. Doing so reveals a 
+drop-down menu full of filter options. From this drop-down, scroll down (or search for) 
+:guilabel:`Type`.
+
+Keep the second portion of the filter equation on the default :guilabel:`= (equal sign)`.
+
+Then, change the third (and final) portion of the filter equation from :guilabel:`Lead` to 
+:guilabel:`Opportunity`. The number of :guilabel:`Records` that fit this specific filter equation 
+will change as the equation is customized.
+
+Add another node to this filter by clicking the :guilabel:`+ (plus sign)` to the right of the 
+equation.
+
+With "new" opportunities being the target of this filter, the second node will focus on only 
+locating opportunities that are in the :guilabel:`New` stage of the pipeline. To do that, select 
+the default :guilabel:`ID` from the first portion of the second filter equation, and scroll down 
+(or search for) :guilabel:`Stage` from the field drop-down menu.
+
+Once again, leave the second portion of the filter equation on :guilabel:`= (equal sign)`.
+
+Lastly, highlight the default value in the third (and final) portion of the second filter equation 
+and type in :guilabel:`New`. With that completed, Odoo will only target opportunities that are in 
+the "New" stage of the pipeline.
+
+.. image:: target_audience/filters-opportunities.png
```

Comment:
Good screenshot, however doesn't need to be so wide.
Suggest replacing with this one: https://i.imgur.com/64Rh14D.png (already cropped + compressed).